src/bgfx.cpp

```diff
@@ -3854,7 +3854,6 @@ namespace bgfx
 
 	void Encoder::blit(ViewId _id, TextureHandle _dst, uint8_t _dstMip, uint16_t _dstX, uint16_t _dstY, uint16_t _dstZ, TextureHandle _src, uint8_t _srcMip, uint16_t _srcX, uint16_t _srcY, uint16_t _srcZ, uint16_t _width, uint16_t _height, uint16_t _depth)
 	{
-		BGFX_CHECK_CAPS(BGFX_CAPS_TEXTURE_BLIT, "Texture blit is not supported!");
```

Comment:
Revert this, this is valid check. If fallback is implemented then just add `BGFX_CAPS_TEXTURE_BLIT` to renderer where fallback is implemented.

Reply:
Agreed, I put it back